### PR TITLE
Added default i18n

### PR DESCRIPTION
**Pull request by ravenjohn**
### Changes
- This is to be able to use `anytv-mailer` without `anytv-i18n`
- Upgraded to `v1.2.0`

### CODE REVIEW

**Review by pandafox — CHANGES_REQUESTED** [review]
The code looks fine but why are we deleting test cases instead of creating another set of test cases specific for none i18n?

**Comment by ravenjohn** [comment]
@pandafox those tests should be in anytv-i18n. the test if the language change is working.

File: src/Templater.js [hunk 1]
```diff
@@ -20,12 +20,53 @@ export default class Templater {
 
         this._built = false;
 
-        this._i18n = config.i18n;
+        this._i18n = config.i18n || this._default_i18n();
```
Comment (pandafox) [inline src/Templater.js]: I think we should change this to transformer or mapper or something so it doesn't need to be handling just i18n. But perhaps any sort of key to value conversions.

**Review by pandafox — CHANGES_REQUESTED** [review]

File: src/Templater.js [hunk 2]
```diff
@@ -20,13 +20,13 @@ export default class Templater {
 
         this._built = false;
 
-        this._i18n = config.i18n || this._default_i18n();
+        this._i18n = config.i18n || this._mapper();
```
Comment (pandafox) [inline src/Templater.js]: How about this variable itself? I guess we need to retain i18n for backward compatibility. but the internal variable can be renamed?

this._mapper = config.mapper || config.i18n || this._mapper();

**Review by pandafox — CHANGES_REQUESTED** [review]

File: src/Templater.js (hunk 2 above)
Comment (ravenjohn) [reply src/Templater.js]: I see..

Comment (ravenjohn) [reply src/Templater.js]: fixed

**Review by pandafox — APPROVED** [review]

File: src/Templater.js [hunk 3]
```diff
@@ -20,16 +20,57 @@ export default class Templater {
...
+
+                let str = key;
+
+                /**
+                 * Replace variables in the string
+                 * `Hello :name!` => `Hello John!`
+                 * `:name aloha!` => `John aloha!`
+                 */
+                _(variables).forOwn((value, _key) => {
+                    str = str.replace(new RegExp(`:${_key}`, 'g'), value);
```
Comment (wildtron) [inline src/Templater.js]: This will incorrectly handle other variables like 
```js
> str = 'This is big :potato_master, but :potato is better'
'This is big :potato_master, but :potato is better'

> str.replace(/:potato_master/g, 'huehue')
'This is big huehue, but :potato is better'
// looks fine

> str.replace(/:potato/g, 'huehue')
'This is big huehue_master, but huehue is better'
// err.. :potato_master shouldn't have been replaced
```

**Review by wildtron — CHANGES_REQUESTED** [review]

File: src/Templater.js (hunk 3 above)
Comment (ravenjohn) [reply src/Templater.js]: ill add `\b`?

**Review by wildtron — APPROVED** [review]
